src/dispatch.rs

```diff
@@ -480,8 +480,8 @@ pub(crate) fn gen_dispatch_str(cmd: DispatchType, dispatch: bool) -> crate::Resu
         MoveToWorkspace(work, None) => format!("movetoworkspace{sep}{work}"),
         MoveToWorkspaceSilent(work, Some(win)) => format!("movetoworkspacesilent{sep}{work},{win}"),
         MoveToWorkspaceSilent(work, None) => format!("movetoworkspacesilent{sep}{work}"),
-        MoveFocusedWindowToWorkspace(work) => format!("workspace{sep}{work}"),
-        MoveFocusedWindowToWorkspaceSilent(work) => format!("workspace{sep}{work}"),
+        MoveFocusedWindowToWorkspace(work) => format!("movetoworkspace{sep}{work}"),
```

Comment:
Instead, maybe we can deprecate this one and keep only `MoveToWorkspace` as it already uses an option for this

Reply:
done (I hope this is what you meant)

Reply:
No use the `#[deprecated]` attribute (also add a reason for deprecation), so it won't break API, but I'll remove it in `0.4` or `1.0`.

Reply:
ah I see, thanks. Deprecated them instead

Cargo now complains about the use of the deprecated field within the lib, when compiling the lib itself; should I silence the warning or leave it as it is?